tools/cldr-apps/js/src/esm/cldrProgress.js

```diff
@@ -188,12 +188,21 @@ function refreshVoterMeter() {
 }
 
 function refreshLocaleMeter() {
-  if (localeProgressStats) {
-    progressWrapper?.updateLocaleMeter(
+  if (localeProgressStats && progressWrapper) {
+    const baselineProblems = localeProgressStats.baselineCount;
+    const currentProblems =
+      localeProgressStats.error +
+      localeProgressStats.missing +
+      localeProgressStats.provisional;
+    const solvedProblems =
+      currentProblems >= baselineProblems
+        ? 0
+        : baselineProblems - currentProblems;
+    progressWrapper.updateLocaleMeter(
       new MeterData(
         cldrText.get("progress_all_vetters"),
-        localeProgressStats.votes,
-        localeProgressStats.total,
+        solvedProblems,
+        baselineProblems,
```

Comment:
I'm thinking we should switch to a better terminology for all 3 meters: "done" and "total".

Here solvedProblems would be "done", and baselineProblems would be "total".

For voter completion and page completion, votes would be "done" and total would be "total".

Reply:
sounds good.